upup/models/cloudup/resources/addons/networking.kuberouter/k8s-1.6.yaml.template

```diff
@@ -153,3 +160,16 @@ subjects:
 - kind: ServiceAccount
   name: kube-router
   namespace: kube-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: system:kube-router
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kube-router
+subjects:
```

Comment:
can add this as a second subject to the existing `kube-router` binding
```
 subjects:
- kind: ServiceAccount
  name: kube-router
  namespace: kube-system
- kind: User
  name: system:kube-router
  apiGroup: rbac.authorization.k8s.io
```

Reply:
@murali-reddy nit pick but do we need `  apiGroup: rbac.authorization.k8s.io
 Write`